### DISCUSSION TITLE
Hi everyone I want to make with nocodb [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5703] [Discord Threads]
> 🙋‍♂️ **romek_rozen**  
#### I want SUM elapsed time on task, person, project in monthly view . It is posibble?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1093596621146706010/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **romek_rozen**  
####   
![image0](https://cdn.discordapp.com/attachments/1093596621339627540/1093597920437215303/image.png)  
#  
> 🙋‍♂️ **romek_rozen**  
####   
![image0](https://cdn.discordapp.com/attachments/1093596621339627540/1093598003119538268/image.png)  
#  
> 💁 **o1lab**  
#### Use rollup  
#  
> 🙋‍♂️ **romek_rozen**  
#### I tried use rollup, but rollup is not getting this field. "czas" is time of one "work" and its calculated, and it is display column  
![image0](https://cdn.discordapp.com/attachments/1093596621339627540/1093798043503050842/image.png)  
![image1](https://cdn.discordapp.com/attachments/1093596621339627540/1093798043750502501/image.png)  
#  
> 🙋‍♂️ **romek_rozen**  
#### Roll-up cant sum from formulas  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093596621339627540)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
